perceiver_pytorch/multi_perceiver_pytorch.py

```diff
@@ -0,0 +1,167 @@
+from perceiver_pytorch.perceiver_pytorch import fourier_encode
+from perceiver_pytorch.perceiver_io import PerceiverIO
+import torch
+from typing import List, Iterable, Dict, Optional, Any
+from einops import rearrange, repeat
+from dataclasses import dataclass
+
+
+@dataclass
+class InputModality:
+    name: str
+    input_channels: int
+    input_axis: int
+    num_freq_bands: int
+    max_freq: float
+    freq_base: int = 2
+    sin_only: bool = False
+    fourier_encode: bool = True
+
```

Comment:
Worth explaining in comments what they all are?

Reply:
Added!